ext/StructArraysTablesExt.jl

```diff
@@ -38,3 +43,5 @@ for (f, g) in zip((:append!, :prepend!), (:push!, :pushfirst!))
         end
```

Comment:
Hm, that's a very good point, we really shouldn't do that. That code may be necessary - we may have to keep Tables as a hard dependency. But maybe Tables itself could be made more lightweight using Pkg extensions or so?

Reply:
Ah, that's a very good point, I hadn't thought about that. Yes, we do need Tables for basic functionality like pushing or appending to a `StructVector` effectively, so that should definitely stay a hard dependency. Still, it is supposed to be an interface package, so that should be OK.